swaybar/render.c

```diff
@@ -435,7 +435,7 @@ static uint32_t render_to_cairo(cairo_t *cairo,
 	}
 	cairo_paint(cairo);
 
-	uint32_t max_height = 0;
+	uint32_t max_height = 1;
```

Comment:
Hmm, this should be already handled by line 488?

Reply:
I added a debug line before line 488 and got `bar->config->height = -1`. In `sway/config/bar.c`, there is `bar->height = -1` (line 82), which is passed across the IPC. Unless `bar <bar-id> height <height>` is set in the config, `bar->config->height` will start off at `-1` and line 488 will not force a positive height. I could change `sway/config/bar.c:82` to `bar->height = 1` instead, if that would be preferred.